x-pack/plugins/alerting/server/routes/run_soon.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { schema } from '@kbn/config-schema';
+import { ILicenseState } from '../lib';
+import { verifyAccessAndContext } from './lib';
+import { AlertingRequestHandlerContext, BASE_ALERTING_API_PATH } from '../types';
+
+const paramSchema = schema.object({
+  id: schema.string(),
+});
+
+export const runSoonRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.post(
+    {
+      path: `${BASE_ALERTING_API_PATH}/rule/{id}/_run_soon`,
```

Comment:
@mikecote Should we make this an internal route to start?

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/40b50cb5653c2e5937115cea9bc2858f4f05cf7a